core/src/main/java/org/fao/geonet/api/records/attachments/Store.java

```diff
@@ -284,6 +284,14 @@ MetadataResource putResource(ServiceContext context, String metadataUuid, String
      */
     String delResources(ServiceContext context, String metadataUuid, Boolean approved) throws Exception;
```

Comment:
Maybe mark this as depreciated as it is possible that the draft metadata uuid is deleted by the time this is called.  So it is prefered to use the function that uses the metadataId

Reply:
yes, we should indeed deprecating this method just like https://github.com/geonetwork/core-geonetwork/blob/4153b0d5e7386defcd4ba50926cac5ecbd41d608/core/src/main/java/org/fao/geonet/api/records/attachments/Store.java#L275-L276

I have commit some more Javadoc and deprecate annotation to it